Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/CriticalCmdletsUsageDetection.yaml

```diff
@@ -0,0 +1,57 @@
+id: 5170c3c4-b8c9-485c-910d-a21d965ee181
+name: VIP Mailbox manipulation
+description: |
+  'High important Cmdlet executed on a VIP Mailbox'
```

Comment:
Wondering if we wanted to rephrase the description to make it more clear what the query is trying to do.

Reply:
Description changed in last push.